ansible/roles/software/ckan/catalog/ckan-app/templates/etc_ckan_production.ini.j2

```diff
@@ -218,7 +218,7 @@ ckanext.spatial.common_map.subdomains = abc
 ckanext.spatial.common_map.attribution = Map tiles &amp; Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="https://creativecommons.org/licenses/by-sa/3.0/">CC BY SA</a>.
 ckanext.spatial.harvest.xslt_html_content = ckanext.datagovtheme:templates/xslt/iso-details.xslt
 ckanext.spatial.harvest.xslt_html_content_original = ckanext.datagovtheme:templates/xslt/esri-iso-fgdc-details.xslt
-ckanext.geodatagov.fgdc2iso_service = http://localhost:8080/fgdc2iso/
+ckanext.geodatagov.fgdc2iso_service = http://{{ catalog_ckan_fgdc2iso_host }}:8080/fgdc2iso/
```

Comment:
https required by bsp?

Reply:
Not by BSP, but by our SSP. All connections should be encrypted.